### DISCUSSION TITLE
zh/code-notes/test-menu2/test2

### DISCUSSION THREAD

**Discussion by giscus** [erqianyi/erqianyi.github.io#2] [Announcements]
# zh/code-notes/test-menu2/test2

前端,前端个人笔记,前端经验分享,javascript,vue,vuepress,vuepress-theme-hope,css

https://2100boy.com/zh/code-notes/test-menu2/test2.html

**Comment by erqianyi** [comment]
测试评论

**Reply by erqianyi** [reply]
移动端测试回复
